packages/eslint-config-sheriff/tsup.config.json

```diff
@@ -8,6 +8,5 @@
   "splitting": false,
   "sourcemap": false,
   "clean": true,
-  "noExternal": ["@sherifforg/constants"],
-  "skipNodeModulesBundle": true
```

Comment:
Does tsup work better now? I'm still concerned that tsup is handling this wrong.

Reply:
im trying different stuff to solve the publishing error. Then running in different errors lol

Reply:
@lishaduck after long exploring, apparently [this thing](https://github.com/egoist/tsup/issues/927#issuecomment-2416440833) is fixing all my issues, and i'm not sure why. But yeah, looks like an open bug on TSUP.

Reply:
What publishing error? The dynamic require error is proof that it's not externalizing node_modules right, and the require thing is weird hack that works right, but means that you'll easily run into the dual package hazard.
The "correct" thing is not to bundle `node_modules`, as ESM code should never need a dynamic require.